templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -3,254 +3,346 @@ mode: 0644
-    # ones, will be run by the runtime_untrusted_workload runtime. Host privileged
-    # containers are by definition trusted and will always use the trusted container
-    # runtime. If default_container_trust is set to "trusted", crio will use the trusted
-    # container runtime for all containers.
-    default_workload_trust = "trusted"
-
-    # no_pivot instructs the runtime to not use pivot_root, but instead use MS_MOVE
-    no_pivot = false
-
-    # conmon is the path to conmon binary, used for managing the runtime.
-    conmon = "/usr/libexec/crio/conmon"
-
-    # conmon_env is the environment variable list for conmon process,
-    # used for passing necessary environment variable to conmon or runtime.
-    conmon_env = [
-      "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-    ]
-
-    # selinux indicates whether or not SELinux will be used for pod
-    # separation on the host. If you enable this flag, SELinux must be running
-    # on the host.
-    selinux = true
+    # The crio.runtime table contains settings pertaining to the OCI runtime used
+    # and options for how to set up and manage the OCI runtime.
+    [crio.runtime]
 
-    # seccomp_profile is the seccomp json profile path which is used as the
-    # default for the runtime.
-    seccomp_profile = "/etc/crio/seccomp.json"
+    # A list of ulimits to be set in containers by default, specified as
+    # "<ulimit name>=<soft limit>:<hard limit>", for example:
+    # "nofile=1024:2048"
+    # If nothing is set here, settings will be inherited from the CRI-O daemon
+    #default_ulimits = [
+    #]
 
-    # apparmor_profile is the apparmor profile name which is used as the
-    # default for the runtime.
-    apparmor_profile = "crio-default"
+    # default_runtime is the _name_ of the OCI runtime to be used as the default.
+    # The name is matched against the runtimes map below.
+    # default_runtime = "runc"
 
-    # cgroup_manager is the cgroup management implementation to be used
-    # for the runtime.
-    cgroup_manager = "systemd"
+    # If true, the runtime will not use pivot_root, but instead use MS_MOVE.
+    # no_pivot = false
 
-    # default_capabilities is the list of capabilities to add and can be modified here.
-    # If capabilities below is commented out, the default list of capabilities defined in the
-    # spec will be added.
-    # If capabilities is empty below, only the capabilities defined in the container json
-    # file by the user/kube will be added.
-    default_capabilities = [
-      "CHOWN", 
-      "DAC_OVERRIDE", 
-      "FSETID", 
-      "FOWNER", 
-      "NET_RAW", 
-      "SETGID", 
-      "SETUID", 
-      "SETPCAP", 
-      "NET_BIND_SERVICE", 
-      "SYS_CHROOT", 
-      "KILL", 
-    ]
+    # Path to the conmon binary, used for monitoring the OCI runtime.
+    # Will be searched for using $PATH if empty.
+    conmon = "/usr/libexec/crio/conmon"
```

Comment:
Does this one still need explicit specification?

Reply:
for the time being yes. This is a situation in which the CRI-O rpm needs to move in lock step with MCO. I guess we can have the rpm put it in both spots (it may already I have to check). Once that happens, we can drop this, and eventually drop it from /usr/libexec all together